pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchFromGitHub, fetchpatch,
-  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales
+  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales, libyubikey, libxi, libxtst, qtx11extras
```

Comment:
the os x failure was unrelated, keepassx-community is not targeted at this platform.